src/browser-frontend/sagas/webcontents-action-handlers.js

```diff
@@ -96,6 +97,10 @@ function* onPageDidNavigateInternal() {
 
 function* onPageDidNavigateToNewWindow({ payload: { parentId, url } }) {
   yield put(PagesModelActions.addPage({ parentId, url, background: false }));
+
+  yield put(PagesModelActions.tabbar.changeOptionalClass({ parentId, optionalClass: 'noanimate' }));
+  yield call(delay, 400);
+  yield put(PagesModelActions.tabbar.changeOptionalClass({ parentId, optionalClass: '' }));
```

Comment:
These three lines are the only thing which aren't 100% obvious with what they do. People like me would probably appreciate at least a short comment describing why optional classes are needed on tabs, and where the '400' magic number is coming from.